storage/remote/otlptranslator/prometheusremotewrite/metrics_to_prw.go

```diff
@@ -26,8 +26,8 @@ import (
 	"go.opentelemetry.io/collector/pdata/pmetric"
 	"go.uber.org/multierr"
 
+	prometheustranslator "github.com/prometheus/otlp-prometheus-translator"
```

Comment:
The linter should have caught it, but this import should be moved to the block starting at line 25, i.e. non-stdlib/non-local imports.